src/clique_csv_writer.erl

```diff
@@ -0,0 +1,92 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2014 Basho Technologies, Inc.  All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+-module(clique_csv_writer).
+
+%% @doc Implements a writer module for clique which outputs tabular data in CSV format.
+
+-behavior(clique_writer).
+
+-export([
+         write/1,
+         write_status/2
+        ]).
+
+-include("clique_status_types.hrl").
+
+-spec write(status()) -> iolist().
+write(Status) ->
+    Output = clique_status:parse(Status, fun write_status/2, []),
+    lists:reverse(Output).
+
+%% @doc Write status information in csv format.
+%%
+%% Anything other than a table is discarded, since there's no good way to represent non-tabular
+%% data in CSV. We want to be able to use the CSV output directly without needing to strip
+%% off any extranious stuff.
+-spec write_status(elem(), iolist()) -> iolist().
+write_status({table, Rows}, Output) ->
+    [write_table(Rows) | Output];
+write_status(_, Output) ->
```

Comment:
Would very much prefer to see alerts sent to stderr than silently swallowed, and I think it makes sense to make that change with this PR so we don't change behavior for operators later.